src/lib/provable/gadgets/gadgets.ts

```diff
@@ -973,6 +974,27 @@ const Gadgets = {
    */
   SHA256: SHA256,
```

Comment:
Should we have a follow-up PR where we get rid of SHA256.ts and use SHA2 instead? The interface could stay the same.

Reply:
I hope so. But there are concerns that this is somehow a "breaking change" and existing zkapps would have to change the syntax to align with this one. Or maybe we can just be backwards compatible in that sense and give an alias for the old name. Either way, this looks like something we could do for a future version of the api.

Reply:
if you add an alias, you can also deprecate the old name with a `@deprecated` annotation